lib/omniauth/auth0/jwt_validator.rb

```diff
@@ -127,7 +127,8 @@ def json_parse(json)
       def uri_string(uri)
         temp_domain = URI(uri)
         temp_domain = URI("https://#{uri}") unless temp_domain.scheme
-        "#{temp_domain}/"
+        temp_domain = "#{temp_domain}/" unless temp_domain.to_s.end_with?('/')
+        temp_domain.to_s
```

Comment:
Maybe do this first so we don't convert to a string twice?

Reply:
Updated